rts/Lua/LuaUnsyncedCtrl.cpp

```diff
@@ -2159,6 +2160,29 @@ int LuaUnsyncedCtrl::SetUnitNoMinimap(lua_State* L)
 	return 0;
 }
 
+/***
+ * @function Spring.SetMiniMapRotation
+ * @number rotation
+ * @treturn nil
+ * @usage Spring.SetMiniMapRotation(0/1/2/3)
```

Comment:
Pls use the new LLS compatible notation here:

```c
 * @function Spring.SetMiniMapRotation
 * @param rotation 0|1|2|3
 * @return nil
 */
```

I would suggest documenting the rotation values too. You could do so similar to [`Facing`](https://github.com/beyond-all-reason/spring/blob/3def0d85b84ce535058e574d3b24c5a8fc740e88/rts/Lua/LuaUnsyncedRead.cpp#L3433-L3439)

So the full thing would be:

```c
/***
 * @alias MiniMapRotation
 * | 0 # Unrotated.
 * | 1 # 90 degrees clockwise.
 * | 2 # 180 degrees.
 * | 3 # 270 degrees clockwise.
 */

/***
 * @function Spring.SetMiniMapRotation
 * @param rotation MiniMapRotation
 * @return nil
 */